modules/probe.sh

```diff
@@ -195,7 +195,12 @@ probe_kernel_in_boot_and_root() {
   done
 
   if [ -z "${kernel_imagetype_path}" ]; then
-    log_warn "Unfortunately we where not able to find the Linux kernel image."
+    log_warn "Unfortunately we were not able to find the Linux kernel image."
+    log_fatal "Please specify the image name using MENDER_GRUB_KERNEL_IMAGETYPE"
+  fi
+
+  if [ "$(echo "${kernel_imagetype_path}" | wc -w)" -gt "1" ]; then
+    log_warn "Unfortunately we were not able to find a unique Linux kernel image."
```

Comment:
There is a separate [PR](https://github.com/mendersoftware/mender-convert/pull/212) to fix this, so would just drop this changeset

Reply:
Yeah, I saw that over the weekend and like the other solution better.